src/apps/website/src/styles/global.css

```
@@ -0,0 +1,519 @@
+  --color-error: #a06050; /* muted red (small concession) */
+
+  --color-nav-bg: rgba(26, 33, 24, 0.95);
+  --color-nav-mobile: rgba(26, 33, 24, 0.98);
+  --color-btn-text: #1a2118; /* dark bg for button labels */
+  --color-code-bg: #1c2127; /* dark charcoal for code blocks */
+  --color-code-header: #212830; /* slightly lighter for headers */
+
+  /* Typography */
+  --font-pixel: "Press Start 2P", monospace;
+  --font-body:
+    "Inter", -apple-system, BlinkMacSystemFont, "Segoe UI", sans-serif;
```

Stylelint (17.5.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)